src/CarbonAware.WebApi/src/api/v1/.gitkeep

```diff
@@ -0,0 +1 @@
+#dummy file created to commit the new folder. Will remove it in the next PR
```

Comment:
What do you expect to go in this folder?

Reply:
The generated swagger.yaml file which contains the api spec